runtime/Cargo.toml

```diff
@@ -104,10 +104,12 @@ pallet-evm-precompile-preimage = { workspace = true }
 substrate-wasm-builder = { workspace = true, optional = true }
 
 [features]
-default = ["std", "with-rocksdb-weights"]
+default = ["std", "with-rocksdb-weights", "mainnet-runtime"]
 with-rocksdb-weights = []
 with-paritydb-weights = []
-fast-runtime = []
+mainnet-runtime = []
+test-runtime = []
```

Comment:
`test-runtime` => `testnet-runtime`
`dev-runtime` => `devnet-runtime`

Just like the `mainnet-runtime`.